### PR TITLE
Milestone 3: Add, Delete and Edit

### PR DESCRIPTION
**Pull request by eroiyo**
Hello!, I have added to remain features of the project, I learned a lot of things with this 📖 

  Added inputs for the descriptions, so you can modify whats are inside them! ✏️ 
    --all the information is saved in localstorage upon change
  
  Created a new file with all the add, remove and edit features 🗃️ 
  
  Now you can submit a task using the mini form at the start of the to-do-list 🔥 
  
  Added focus and off focus events for the task 👓

### CODE REVIEW

File: src/addEditErase.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  }, 200);
+}
+export function offfocus(element, handle, trashcan) {
+  setTimeout(() => {
+    element.parentNode.parentNode.classList.remove('focused');
+    element.classList.remove('focused');
+    handle.classList.remove('hidden');
+    trashcan.classList.add('hidden');
+  }, 200);
+}
```
Comment (carlos-ssh) [inline src/addEditErase.js:77]: **Kindly fix these errors:** 

- Kindly fix how to create a new To Do, Clear all.
- [OPTIONAL LINE]Add into your .gitignores not related project files like: .vsvode, .DS_Store.
Screen Shot 2021-07-17 at 08 12 25

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @eroiyo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

_I followed the installation steps that you specified into your README file; it runs but i can't use your application.
Kindly check the errors commented._


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Add into your .gitignores not related project files like: .vsvode, .DS_Store.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addEditErase.js (hunk 1 above)
Comment (eroiyo) [reply src/addEditErase.js:77]: Thanks, i did not program a case of local storage being null, i will fix that! 🔧

Comment (carlos-ssh) [reply src/addEditErase.js:77]: 🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽🙌🏽

**Comment by eroiyo** [comment]
Ready for another Review!

**Comment by carlos-ssh** [comment]
> Ready for another Review!

Send it into your dashboard!!

**Review by dasileker — APPROVED** [review]
### **Hello @eroiyo** 

## Project Approved ✅✅

Your project is complete!  There is nothing else to say other than... it's time to merge it✈✈⛴

**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard).

**Comment by eroiyo** [comment]
Thanks Reviewers!

**Pull request merged**